packages/cpp/ArmoniK.Api.Client/header/versions/VersionsClient.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "versions_common.pb.h"
+#include "versions_service.grpc.pb.h"
+
+namespace armonik {
+namespace api {
+namespace client {
+
+/**
+ * Versions Client wrapper
+ */
+class VersionsClient {
+public:
+  explicit VersionsClient(std::unique_ptr<armonik::api::grpc::v1::versions::Versions::StubInterface> stub)
+      : stub(std::move(stub)){};
+
+  /**
+   * Get versions of ArmoniK components
+   * @return Mapping between component names and their versions
+   */
+  std::map<std::string, std::string> list_versions();

```

Comment:
Why `std::map` instead of `std::unordered_map` ? Is it because of the `std::string`'s hash ?

Reply:
@ddiakiteaneo You should define a struct here.